interface/src/app/gene/network/network.js

```diff
@@ -130,16 +131,17 @@ angular.module('adage.gene.network', [
         } else {
           correlationSign = 1;
         }
-        network.filterEdgeWeight(self.slider.min, maxCorrelation,
-                                 correlationSign);
+        network.filterWithWeightSign(
+          self.minEdgeWeightSlider.value, maxCorrelation,
+          self.maxGeneNumSlider.value, correlationSign);
```

Comment:
I used this name mainly because Aaron had one function just called `filter` in line 261 of `d3.network.js`. The only difference between `filter` and this one is that the former doesn't consider the sign, but the latter does.

Reply:
If the naming (and parameter order) makes sense in context then it's fine with me. As you said, we aren't really reviewing the `d3.network.js` file here and I think we have a good plan for getting it reviewed moving forward.

Reply:
Tagging issue #295 here to link it to this set of comments.